examples/Superbible/Vertex Colouring/.gitignore

```diff
@@ -0,0 +1,3 @@
+obj
```

Comment:
`obj` is already ignored by the top level `.gitignore` and does not need to be repeated here.